seed/challenges/02-javascript-algorithms-and-data-structures/basic-data-structures.json

```diff
@@ -394,18 +435,64 @@
       "translations": {}
     },
     {
+      "id": "587d7dac367417b2b25184d3",
+      "title": "Basic Data Structures: Introduction to Objects",
+      "description": [
+        [
+          "",
+          "",
+          "The next data structure we will discuss is the JavaScript <dfn>object</dfn>. Like arrays, objects are a very fundamental part of JavaScript. However, it is probably safe to say that objects surpass arrays in flexibility, usefuleness and in their overall importance to the language &mdash; in fact, you may have heard this line before: 'In JavaScript, everything is an object.'<br><br>While an understanding of objects is important to understand the inner workings of JavaScript functions or JavaScript's object-oriented capabilities, JavaScript objects at a basic level are actually just <dfn>key-value pair</dfn> stores, a commonly used data structure across almost all programming languages. Here, we will confine our discussion to JavaScript objects in this capacity.",
```

Comment:
Remove "very" in "objects are a very fundamental". Also, fix typo "usefuleness" should be "usefulness".